spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
`[partner]` is not something that could be sent to the view.  Exception `undefined method 'total_pages' for ...`  So I'm not sure if using it in the test makes sense.  It would be just as appropriate to return some random object, as it would to return an array or AR_relation.  The view cannot consume an array.